Packs/CrowdStrikeFalcon/Integrations/CrowdStrikeFalcon/CrowdStrikeFalcon.yml

```diff
@@ -3595,7 +3604,787 @@ script:
+      type: Number
+    - contextPath: CrowdStrike.ODSScan.scan_started_on
+      description: The timestamp when the scan was started.
+      type: Date
+    - contextPath: CrowdStrike.ODSScan.scan_completed_on
+      description: The timestamp when the scan was completed.
+      type: Date
+    - contextPath: CrowdStrike.ODSScan.created_on
+      description: The timestamp when the scan was created.
+      type: Date
+    - contextPath: CrowdStrike.ODSScan.created_by
+      description: The ID of the user who created the scan job.
+      type: String
+    - contextPath: CrowdStrike.ODSScan.last_updated
+      description: The timestamp when the scan job was last updated.
+      type: Date
+  - arguments:
+    - description: List of host groups to be scanned.
+      isArray: true
+      name: host_groups
+      required: true
+    - description: List of file paths to be scanned. "file_paths" OR "scan_inclusions" must be set.
+      isArray: true
+      name: file_paths
+      required: false
+    - description: List of included files or locations for this scan. "file_paths" OR "scan_inclusions" must be set.
+      isArray: true
+      name: scan_inclusions
+    - description: List of excluded files or locations for this scan.
+      isArray: true
+      name: scan_exclusions
+      required: false
+    - description: Scan origin.
+      name: initiated_from
+      required: false
+    - description: Set the scan CPU priority.
+      name: cpu_priority
+      required: false
+      auto: PREDEFINED
+      defaultValue: Low
+      predefined:
+      - Highest
+      - High
+      - Medium
+      - Low
+      - Lowest
+    - description: Scan description.
+      name: description
+      required: false
+    - description: Flag indicating if identified threats should be quarantined.
+      name: quarantine
+      required: false
+    - description: Amount of time (in seconds) for scan pauses.
+      name: pause_duration
+      required: false
+      defaultValue: '2'
+    - description: Sensor ML detection level.
+      name: sensor_ml_level_detection
+      required: false
+    - description: Sensor ML prevention level.
+      name: sensor_ml_level_prevention
+      required: false
+    - description: ML Detection level for the scan.
+      name: cloud_ml_level_detection
+      required: false
+    - description: ML Prevention level for the scan.
+      name: cloud_ml_level_prevention
+      required: false
+    - description: Maximum time (in seconds) the scan is allowed to execute.
+      name: max_duration
+      required: false
+      defaultValue: '2'
+    - description: When to start the first scan. Supports english expressions such as "tommorow" or "in an hour".
+      name: schedule_start_timestamp
+      required: true
+    - description: Set the schedule interval.
+      name: schedule_interval
+      required: true
+      auto: PREDEFINED
+      predefined:
+      - never
```

Comment:
lets have these start with caps. We also shouldtnt use numbers. Every four weeks

Reply:
done